packages/strapi-design-system/package.json

```
@@ -61,6 +61,7 @@
     "build": "yarn build:prod",
     "build:prod": "vite build",
     "generate:types": "tsc --noEmit false --emitDeclarationOnly --declarationDir dist",
+    "lint": "eslint . --ext .js,.jsx,.tsx,.ts --fix",
```

Should we be consistent with the CMS here and split `linting` from `fixing`, by setting up two tasks (e.g. for CI)?

```suggestion
    "lint": "eslint . --ext .js,.jsx,.tsx,.ts --fix",
    "format": "yarn lint -- --fix",
```

---

i only kept fix cause it was there, to be honest i don't mind omitting it and making the developer run the fix command lol. Your suggestion seems off so I can't commit it by the way.

---

What do you mean by "off"?

---

You didn't remove `--fix` 👀  sneaky

---

Yes, I'm guilty of editing the message afterward. Sorry 🙈